scripts/start-formatter-lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pre-commit run --all-files
```

Comment:
**nitpick(non-blocking):** I'd add the command `git config --global --add safe.directory /app` case someone might want to run it through a compose service.